src/openforms/templates/master.html

```diff
@@ -9,17 +14,28 @@
     <link href="{% static 'bundles/core-css.css' %}" media="all" rel="stylesheet"/>
     <link href="{% static 'hijack/hijack-styles.css' %}" rel="stylesheet" type="text/css"/>
     {% block extra_css %}{% endblock %}
+    {% if enable_analytics %}
+        {# Load the various supported analytics provider snippets #}
+        {% include "includes/analytics/all_head.html" %}
+    {% endif %}
 </head>
 
 <body class="view__body">
-{% hijack_notification %}
+    {% hijack_notification %}
+
+    {% block content %}
+        Override <code>{% verbatim %}{% block content %}{% endverbatim %}</code>
+        to get some actual content.
+    {% endblock %}
 
-{% block content %}
-    Override <code>{% verbatim %}{% block content %}{% endverbatim %}</code>
-    to get some actual content.
-{% endblock %}
+    {% include "includes/cookie-notice.html" %}
 
-<script src="{% static 'bundles/openforms-js.js' %}"></script>
-{% block extra_js %}{% endblock %}
+    <script src="{% static 'bundles/openforms-js.js' %}"></script>
+    {% block extra_js %}{% endblock %}
+    {% if enable_analytics %}
+        {# Load the various supported analytics provider snippets #}
+        {% include "includes/analytics/all_bottom.html" %}
+    {% endif %}
```

Comment:
Although this is a temporary page, we should include a link to the cookie setting page at all times (footer stuff prolly). Something not to forget when we create the subdomain page.